### DISCUSSION TITLE
I Use Arch BTW Edition

### DISCUSSION THREAD

**Discussion by excalith** [excalith/excalith-start-page#28] [Showcase]
Here is a screenshot with a custom wallpaper and fetch image on top of default configuration using `synthwave` theme

<img width="1273" alt="List Screen" src="https://user-images.githubusercontent.com/9248355/231784932-4edc9916-5e8b-4b94-97ff-813c456022d5.png">

<img width="1273" alt="Fetch Screen" src="https://user-images.githubusercontent.com/9248355/231787004-f03f3b4f-935b-431e-b730-ea721ed590ef.png">

**Comment by TheLinuxGuy-ssh** [comment]
![image](https://user-images.githubusercontent.com/107614947/232314494-fdbb863f-16c0-4c51-9d89-1ceb942a5743.png)
I do like but how about this....
Did this with Tokyo Night Theme...

**Reply by excalith** [reply]
Hey, looks great! Loved the logo and the glow. Might add this as a feature to place logo somewhere using the same glow fx with terminal. Cheers!

**Reply by TheLinuxGuy-ssh** [reply]
Didn't quite understand what you meant by that...

**Reply by excalith** [reply]
I meant the logo on the top center is a nice touch. I may implement it as a feature to add logo from configuration file, so no need to embed it into the wallpaper.

**Reply by TheLinuxGuy-ssh** [reply]
I see, but how do i edit the files? i cloned it from docker and i am new to linux, started out 1-2 months ago....

**Reply by excalith** [reply]
Hey @Rohan-Developer0 can you please use Q&A discussions for questions? If someone else needs to ask, they can find it there instead of this showcase screenshot. While writing though, can you please elaborate more? What do you want to do and which files you want to change?

And instead of replying in this topic, you can post your screenshot as a new Showcase within discussions as well

**Comment by ParadoxPacer** [comment]
That looks great! How did you get the Terminal to glow like that? I can't seem to replicate it.

**Reply by excalith** [reply]
Hey mate, glad to hear it! 

I believe what you are looking for are located in `terminal` section inside the config. 

```
"terminal": {
   "fixedHeight": true,  // Locks window height to list view (other views such as help won't change the height)
   "windowGlow": true,   // Enables or disables terminal window border glow
   "windowGlowColor": "#291f3325", // Sets terminal window border glow color
   "textGlow": false,    // Enables text glow
   "opacity": 1,         // Changes terminal window opacity (Between 0 and 1)
   "blur": 0             // Sets terminal background blur to given integer in pixels (Works best with wallpaper and overrides both terminal opacity  and terminal background color)
}
```

Please refer this [wiki page](github.com/excalith/excalith-start-page/wiki/Configuration#terminal) for more info

**Reply by ParadoxPacer** [reply]
I think theres an error with the color code, i shortened the windowGlowColor value to #291f33 and it worked.
